File: backEnd/gerenciadorEstoque/src/main/java/by/gabriel/gerenciadorEstoque/Api/DTO/Movimentacao/MovimentacaoDTO.java
package by.gabriel.gerenciadorEstoque.Api.DTO.Movimentacao;

import by.gabriel.gerenciadorEstoque.Enum.Movimentacao.AcaoMovimentacao;
import by.gabriel.gerenciadorEstoque.Enum.Movimentacao.TipoEntidade;
import java.time.LocalDateTime;

public record MovimentacaoDTO(
        Long movId,
        TipoEntidade tipoEntidade,
        String registroAfetadoId,
        String nomeRegistroAfetado,
        AcaoMovimentacao acaoMov,
        String campoAfetado,
        LocalDateTime dataMov,
        String responsavel
) {}

File: frontEnd/estoque-vendas-main/src/app/models/movimentacaoDTO.ts
export interface movimentacaoDTO { // <-- Só colocar o M maiúsculo aqui
  movId: number;
  tipoEntidade: string;         
  registroAfetadoId: string;
  nomeRegistroAfetado: string;  
  acaoMov: string;
  campoAfetado: string;
  dataMov: string;
  responsavel: string;
}